toolbox/index.rst

```diff
@@ -0,0 +1,31 @@
+Hacker's Toolbox
+================
+
+Lots of very smart people start as a CS major with little to no programming
+experience. Then they stumble upon HacSoc and hear all these people talking
+about things they may not know about (Linux, git, about a hundred different
+programming languages, etc). It can be really intimidating.
+
+**If you are in this situation, know that you aren't behind at all!**
+
+We learn these things through time and experience. Classes don't teach them all.
+Some classes may even expect Linux, C, or Git experience, without even having
+taught you them before. Skills like these are very practical, but they're not
+exactly what "Computer Science" is about. They're all just part of the hacker's
+toolbox.
+
+This section is an intro to the sorts of practical tools you may find yourself
+needing in classes or the real world. While we can't promise to have complete
+information, the goal is to have enough to get you started and understand how to
+learn more on your own. You can *always* ask for help on Slack or the mailing
+list.
+
+*Recommended way to go through this section:* Follow through the first few
+articles, up to Git. From there, see about doing your Java or Data Structures
+homework on Linux! Once you're comfortable doing this, you can even begin
```

Comment:
nit: remove "even" at the end of the line? so "you can even begin" -> "you can begin"

Reasoning: I feel that the use of the word "even" makes it seem like the thing you are beginning (learning other languages and technologies) is more daunting than without the word "even."